### PR TITLE
Add translation section

### PR DESCRIPTION
**Pull request by sanderfoobar**
As self-hosted translation solutions are getting quite good nowadays, it is reasonable to dedicate a new section to it. So, this PR introduces the **Translation** section and adds 1 project (and migrates 1 project from Miscellaneous).

As per this PR, the translation section would have:

- [Kotki](https://github.com/kroketio/kotki)
- [Libre Translate](https://libretranslate.com/)

(Disclaimer: I am the author of Kotki, the self-hosted bergamot based translation engine)

### Checklist

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by KieranRobson** [comment]
@sanderfoobar Could you add a section description, similar to the other sections, normally from Wikipedia such as

[Translation]() software is blah blah

**Review by nodiscc — CHANGES_REQUESTED** [review]
> Translate Locally is a desktop application

>> This is a list of [Free](https://en.wikipedia.org/wiki/Free_software) Software [network services](https://en.wikipedia.org/wiki/Network_service) and [web applications](https://en.wikipedia.org/wiki/Web_application) which can be hosted on your own server(s).

Please remove Translate Locally.

File: README.md [hunk 1]
```diff
@@ -1992,6 +1992,14 @@ _Related: [Task Management & To-do Lists](#task-management--to-do-lists), [Softw
 - [Kimai](https://www.kimai.org/) - Kimai is a free & open source timetracker. It tracks work time and prints out a summary of your activities on demand. ([Demo](https://www.kimai.org/demo/), [Source Code](https://github.com/kevinpapst/kimai2/)) `MIT` `PHP`
 - [TimeTagger](https://timetagger.app) - An open source time-tracker based on an interactive timeline and powerful reporting. ([Demo](https://timetagger.app/app/demo), [Source Code](https://github.com/almarklein/timetagger)) `GPL-3.0` `Python`
 
+### Translation
+
+**[`^        back to top        ^`](#awesome-selfhosted)**
+
+[Machine translation](https://en.wikipedia.org/wiki/Machine_translation) translates text from one language to another via corpus statistical or neural network techniques.
+
+- [Kotki](https://github.com/kroketio/kotki) - Bergamot based language translations, entirely self-hosted. HTTP API, CLI, and library. ([Demo](https://kotki.kroket.io)) `MPL-2.0` `Python / C++`
```
Comment (nodiscc) [inline README.md:2001]: > https://github.com/kroketio/kotki#requirements
>> Get MKL installed.
>>> https://en.wikipedia.org/wiki/Math_Kernel_Library
>>> License: [freeware](https://en.wikipedia.org/wiki/Freeware) under [ISSL](https://www.intel.com/content/www/us/en/developer/articles/license/end-user-license-agreement.html#collapseCollapsible1665000686280)
>>>> No reverse engineering, decompilation, or disassembly of the Software is permitted, nor any modification or alteration of the Software or its operation at any time, including during execution.

> https://en.wikipedia.org/wiki/Free_software
>> Freedom 1: The freedom to study how the program works, and change it to make it do what you wish.

This must be moved to non-free.md and ISSL added to the license list there.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — CHANGES_REQUESTED** [review]
> hacked Intel MKL out of the codebase and replaced it for OpenBLAS (FOSS alternative) - should be 100% FOSS now

Nice!

>  Any category you are creating has the minimum requirement of 3 items.
> If not, your addition may be inserted into Misc/Other.

Please move these 2 items to `Miscellaneous` (or find a third machine translation tool to add to the new section)

**Comment by nodiscc** [comment]
@sanderfoobar ping

**Comment by zen0bit** [comment]
Maybe also:
- [Apertium](https://www.apertium.org) - Shallow-transfer machine translation engine, capable of performing translations between related languages. ([Source Codes](https://github.com/apertium)) `GPL-3.0` `C++`
- [OpenNMT](https://opennmt.net) - Neural machine translation system based on the Torch library, capable of performing translations between many different language pairs. `MIT` `Lua/Python`

**Comment by nodiscc** [comment]
No reply for > a month, closing. Freel free to open a new Pull Request.

**Pull request closed without merging**